frontend/src/components/App/Home/SquareButton.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { Meta, Story } from '@storybook/react/types-6-0';
+import SquareButton, { SquareButtonProps } from './SquareButton';
+
+export default {
+  title: 'common/SquareButton',
+  component: SquareButton,
+  argTypes: {},
+} as Meta;
+
+const Template: Story<SquareButtonProps> = args => <SquareButton {...args} />;
+
+export const Basic = Template.bind({});
+Basic.args = {
+  icon: 'mdi:pencil',
+  label: 'Some label',
+};
```

Comment:
There could be more states to cover these properties:
-   iconSize?: number;
-   iconColor?: string;
-   primary?: boolean;